BaseClasses.py

```diff
@@ -1733,3 +1747,83 @@ def get_seed(seed: Optional[int] = None) -> int:
         random.seed(None)
         return random.randint(0, pow(10, seeddigits) - 1)
     return seed
+
+
+# These are tags that appear commonly in worlds, but you are not limited to only these choices.
+# It'd be nice to standardize these so worlds don't have to do fuzzy matching on item names or similar,
+# but I don't think these should be all-encompassing.
+class CommonVisualTags:
```

Comment:
"Money" and "equipment" might be common enough kinds of items to warrant inclusion here.